xds/src/main/java/io/grpc/xds/internal/sds/XdsServerBuilder.java

```diff
@@ -174,7 +161,7 @@ void panic(final Throwable t) {
           });
     InternalProtocolNegotiator.ProtocolNegotiator serverProtocolNegotiator =
         SdsProtocolNegotiators.serverProtocolNegotiator(
-            this.downstreamTlsContext, port, syncContext);
+            port, syncContext);
```

Comment:
this looks like it can be 1 line.

Reply:
formatted by my IDE :-) but will make the change